plugins/woocommerce/src/Admin/Features/ProductBlockEditor/Init.php

```diff
@@ -240,15 +240,33 @@ private function get_default_product_templates() {
 		$templates   = array();
 		$templates[] = new ProductTemplate(
 			array(
-				'id'                 => 'standard-product-template',
-				'title'              => __( 'Standard product', 'woocommerce' ),
-				'description'        => __( 'A single physical or virtual product, e.g. a t-shirt or an eBook.', 'woocommerce' ),
-				'order'              => 10,
-				'icon'               => 'shipping',
-				'layout_template_id' => 'simple-product',
-				'product_data'       => array(
+				'id'                      => 'standard-product-template',
+				'title'                   => __( 'Standard product', 'woocommerce' ),
+				'description'             => __( 'A single physical or virtual product, e.g. a t-shirt or an eBook.', 'woocommerce' ),
+				'order'                   => 10,
+				'icon'                    => 'shipping',
+				'layout_template_id'      => 'simple-product',
+				'product_data'            => array(
 					'type' => 'simple',
 				),
+				'alternate_product_datas' => array(
+					array(
+						'type' => 'variable',
+					),
+				),
+			)
+		);
+		$templates[] = new ProductTemplate(
```

Comment:
It's taking me a bit of time to wrap my head around why variations need their own templates since they also share the same template with "Standard product," right?

It seems like we might be moving back towards a 1:1 for templates to product types (not saying that's necessarily a bad thing), just noting that we now have product templates, layout templates, and types for each.  It does feel like there might be room to simplify things a bit.

Reply:
`variable` products and `simple` products both are "Standard Products". As such, they share a single product template (and therefore the `simple-product` layout template).

Variations are handled specially in the product editor (both the new product editor and the classic editor). You can't create them outside of the context of a `variable` product. They have no product `type`. And, they use a different layout template (`product-variation`). I do think that they could probably use the same `simple-product` layout template, with `hideConditions`, but I think the implementation of variations predated the conditionally visibility support... I can't remember for sure.